brave-core/src/main/java/com/github/kristofa/brave/ClientTracer.java

```diff
@@ -37,7 +36,7 @@ public static Builder builder() {
     abstract ClientSpanAndEndpoint spanAndEndpoint();
     abstract Random randomGenerator();
     abstract SpanCollector spanCollector();
-    abstract List<TraceFilter> traceFilters();
```

Comment:
no more list.. if people have cause to create multiple tiers of filtering, they can make a composite one.